gradle/dependencies.gradle

```diff
@@ -177,7 +177,6 @@ libs += [
   kafkaStreams_28: "org.apache.kafka:kafka-streams:$versions.kafka_28",
   kafkaStreams_30: "org.apache.kafka:kafka-streams:$versions.kafka_30",
   kafkaStreams_31: "org.apache.kafka:kafka-streams:$versions.kafka_31",
-  log4j: "log4j:log4j:$versions.log4j",
```

Comment:
Can we just switch this to point to reload4j?

Reply:
I've kept things in alphabetical order

Reply:
Yip that was simpler